packages/rest/src/types.ts

```diff
@@ -225,3 +225,6 @@ export interface Session {
 export interface RequestWithSession extends Request {
   session: Session;
 }
+
+// For backwards compatibility
```

Comment:
👍🏻 

Perhaps add a `TODO(SEMVER-MAJOR)` comment to make it easy to find this opportunity for backwards-incompatible cleanup when working on a new major version?